watchtower-plugin/src/retrier.rs

```diff
@@ -36,10 +36,15 @@ impl Retrier {
 
         loop {
             let tower_id = unreachable_towers.recv().await.unwrap();
-            self.wt_client
-                .lock()
-                .unwrap()
-                .set_tower_status(tower_id, crate::TowerStatus::TemporaryUnreachable);
+            {
+                // Not start a retry if the tower is flagged to be abandoned
+                let mut wt_client = self.wt_client.lock().unwrap();
+                if wt_client.towers.get(&tower_id).is_none() {
+                    log::info!("Abandoning {}", tower_id);
```

Comment:
Can this be like "skipping abandoned tower {}" instead. since the tower is already abandoned at this point.

Reply:
sorry, "skipping retrying abandoned tower {}"* is better